LayoutTests/platform/wk2/TestExpectations

```diff
@@ -706,6 +706,7 @@ fast/images/gif-loop-count.html [ Pass ]
 
 # Only relevant for WebKit2
 http/tests/storage/storage-map-leaking.html [ Pass ]
+http/tests/notifications/permission/ [ Pass ]
```

Comment:
Please move this right next to the following line in this file:
`imported/w3c/web-platform-tests/permissions/ [ Pass ]`

Reply:
Fixed!